.circleci/config.yml

```diff
@@ -381,11 +372,11 @@ workflows:
           type: approval
           requires:
             - android-integration-test-build
-          <<: *release-branches-and-main
+          <<: *release-branches
       - run-firebase-tests:
           requires:
             - hold
-          <<: *release-branches-and-main
+          <<: *release-branches
```

Comment:
Hmm I think we should probably just remove this. Note that we are already running the integration tests on release branches as part of the `deploy` workflow. IMO, I think this should be running in the `main` branch only. Thoughts?